python/ray/rllib/agent.py

```diff
@@ -240,6 +240,9 @@ def get_agent_class(alg):
     elif alg == "PG":
         from ray.rllib import pg
         return pg.PGAgent
+    elif alg == "DDPG":
+        from ray.rllib import ddpg
+        return ddpg.DDPGAgent
```

Comment:
Nice. We should also make an issue to update the docs; there are a couple new algs not documented.

Reply:
@alvkao58 can you take care of these?

Reply:
any particular sanity checks you want to see added to multi_node_tests.sh?

Reply:
any particular sanity checks you want to see added to multi_node_tests.sh?